### PR TITLE
Update trace-integration.md

### PR DESCRIPTION
**Pull request by samu6851**
The link listed for Trace to metrics takes you to the home page for the tempo datasource.  The link I have provided takes you to the trace to metrics section of the configure Tempo page for the Temp datasource.  

This should make it much easier for customers as they will be take directly to the section on this feature.

**What is this feature?**

[Add a brief description of what the feature or update does.]

**Why do we need this feature?**

[Add a description of the problem the feature is trying to solve.]

**Who is this feature for?**

[Add information on what kind of user the feature is for.]

**Which issue(s) does this PR fix?**:

Fixes #

**Special notes for your reviewer:**

Please check that:
- [ ] It works as expected from a user's perspective.
- [ ] If this is a pre-GA feature, it is behind a feature toggle.
- [ ] The docs are updated, and if this is a [notable improvement](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/#how-to-determine-if-content-belongs-in-a-whats-new-document), it's added to our [What's New](https://grafana.com/docs/writers-toolkit/writing-guide/contribute-release-notes/) doc.

### CODE REVIEW

File: docs/sources/explore/trace-integration.md [hunk 1]
```diff
@@ -115,7 +115,7 @@ Click the document icon to open a split view in Explore with the configured data
 This feature is currently in beta & behind the `traceToMetrics` feature toggle.
 {{% /admonition %}}
 
-You can navigate from a span in a trace view directly to metrics relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](/docs/grafana/latest/datasources/tempo/#trace-to-metrics) for configuration instructions.
+You can navigate from a span in a trace view directly to metrics relevant for that span. This feature is available for Tempo, Jaeger, and Zipkin data sources. Refer to their [relevant documentation](https://grafana.com/docs/grafana/latest/datasources/tempo/configure-tempo-data-source/#trace-to-metrics) for configuration instructions.
```
Comment (gelicia) [inline docs/sources/explore/trace-integration.md:118]: there are relative URLs all over the place in this file, I feel like you should have this one be relative too

**Review by gelicia — CHANGES_REQUESTED** [review]

File: docs/sources/explore/trace-integration.md (hunk 1 above)
Comment (samu6851) [reply docs/sources/explore/trace-integration.md:118]: Sure I will be happy to make this a relative link, I just need to know how to do that.  Sorry, I am not familiar with this process :)

Comment (gelicia) [reply docs/sources/explore/trace-integration.md:118]: Ah, there's no process. Look at the diff - what was there before was a relative link, and by adding the protocol and domain, it's an absolute link. You should just have to remove `https://grafana.com` from the URL and be good to go

Comment (samu6851) [reply docs/sources/explore/trace-integration.md]: Cool, thanks for letting me know.  I have made this change.

**Review by gelicia — APPROVED** [review]

**Review by aocenas — APPROVED** [review]

**Pull request merged**